app/cache/CacheRepository.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cache
+
+import play.api.Configuration
+
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class CacheRepositoryConfiguration @Inject() (configuration: Configuration) {
```

Comment:
Can we make sure file name matches to the class here.
Also, is this actually used? I don't see any usages of this configuration fields. Looks like we can drop it